pkg/network/node/sdn_controller.go

```diff
@@ -78,12 +79,12 @@ func (plugin *OsdnNode) alreadySetUp(localSubnetGatewayCIDR string, clusterNetwo
 		}
 	}
 	if !found {
-		return false
+		return errors.New("Local subnet gateway CIDR not found")
```

Comment:
Errors are usually lowercase (as @pravisankar keeps reminding me)

Same for the following errors.

Reply:
Do you mean that they should start with a lowercase letter like this? "local subnet gateway CIDR not found"

Reply:
yes